fe/fe-core/src/main/java/com/starrocks/persist/OperationType.java

```diff
@@ -216,4 +216,6 @@ public class OperationType {
     // drop external catalog
     public static final short OP_DROP_CATALOG = 10061;
 
+    public static final short OP_CREATE_INSERT_OVERWRITE = 10070;
```

Comment:
Do we have comments to explain the difference and connection between these two states ？ in the process of reviewing the code, I can't simply understand their meaning from the name

Reply:
OP_CREATE_INSERT_OVERWRITE is for creation of insert overwrite job
OP_INSERT_OVERWRITE_STATE_CHANGE is for state change of insert overwrite job.

Reply:
PTAL